OpenRA.Platforms.Default/Shader.cs

```diff
@@ -151,6 +155,7 @@ public void PrepareRender()
 			OpenGL.glUseProgram(program);
 			OpenGL.CheckGLError();
 
+			// Future notice: using ARB_bindless_texture makes all the gl calls below obsolete.
```

Comment:
This is a GL4 feature so i'm not sure we will ever be able to use it.

Reply:
Acceptable but sad :( might be worth however to make that somehow mod implementable in the future

Reply:
Been thinking about this: i would suggest to leave this in. We can simply check whether this extension is available or not. As soon as it is, we can implement that into the shaders so they use the feature if present, but not if not present. I've opened a new github repo where i play with a renderer which i try to be able to render a full scene in a single GL call. Basically im aiming for the absolute perfect scenario, then adding all the checks and workarounds for when a specific feature is not available. When being back at OpenRA's gl version, it could be a perfect candidate to implement as renderer.